api/models/call.go

```diff
@@ -146,9 +136,16 @@ type Call struct {
 	Error string `json:"error,omitempty" db:"error"`
 }
 
+// Call is a representation of a specific invocation of a route.
+type Call struct {
+	CallBase
```

Comment:
could you undo this? it seems only to add bloat and is a hassle for all sites needing to build a call. there is only one place i see a call constructed without the app id, which is in tests, and doesn't seem that hard to fix that up.

Reply:
(not worth it for one field)

Reply:
to be clear, just put all the fields back on `Call`

Reply:
actually, it's not true, call without app ID also used at the API level, so the user wouldn't see the app ID.

Reply:
but yeah, we can just hide that